### ISSUE TITLE
Cache views for re-use

### ISSUE THREAD

**Issue by anzorb** [pwalczyszyn/backstack#17]
I think it will be useful to have some kind of caching, to re-use already rendered views. For example, I use the following function to cache my templates, maybe you can add something similar? (of course minus the template stuff). Great job on the window stack btw!

```
var TemplateCache = {
          get: function(id,view){
            if (!this.templates){ this.templates = {}; }

            var template = this.templates[id];
            if (!template){
              template = view;

              // precompile the template, for underscore.js templates
             // template = _.template(template);

              this.templates[id] = template;
            }

            return template;
          }
}
```

I then call this.$el.append(TemplateCache.get("#list-item"));

**Comment by anzorb** [comment]
Sorry, let me clarify, it seems that when going back to pages already visited, they are re-rendered. Is there a way to cache the renders? Thanks!

**Comment by joelso** [comment]
Also struggled with this and saw that it has already been addressed here #12 

If you add "destructionPolicy" to your backbone view it will be reused when it's pushed/popped:

```
var view = Backbone.View.extend({
    // ...
    destructionPolicy: 'never'
    // ...
})
```

**Comment by anzorb** [comment]
Thanks joelso! Closing issue.
